test/functional/filters/multilineexception/multiline_exception_test.go

```diff
@@ -116,57 +113,18 @@ created by main.main
 		Entry("of V8 errors stack trace", jsV8Exception, nil),
 		Entry("of NodeJS services", nodeJSException, nil),
 		Entry("of GoLang services", goLangException, nil),
-		Entry("of single application NS to single pipeline", goLangException, func(framework *functional.CollectorFunctionalFramework) {
-			Skip("TODO: FIX MEskip for vector")
-			testruntime.NewClusterLogForwarderBuilder(framework.Forwarder).
-				FromInputName("forward-pipeline", func(spec *obs.InputSpec) {
-					spec.Type = obs.InputTypeApplication
-					spec.Application = &obs.Application{
-						Includes: []obs.NamespaceContainerSpec{
-							{Namespace: appNamespace},
-						},
-					}
-				}).
-				WithMultineErrorDetection().
-				ToHttpOutput()
-		}),
 		Entry("of single application NS sources with multiple pipelines", goLangException, func(framework *functional.CollectorFunctionalFramework) {
-			Skip("TODO: FIX MEskip for vector")
-			b := testruntime.NewClusterLogForwarderBuilder(framework.Forwarder).
-				FromInputName("multiline-log-ns", func(spec *obs.InputSpec) {
-					spec.Type = obs.InputTypeApplication
-					spec.Application = &obs.Application{
-						Includes: []obs.NamespaceContainerSpec{
-							{Namespace: appNamespace},
-						},
-					}
-				}).
-				WithMultineErrorDetection().
-				ToHttpOutput()
-			//LOG-2241
-			b.FromInput("multiline-log-ns").
-				Named("other").
-				WithMultineErrorDetection().
-				ToElasticSearchOutput()
-		}),
-		Entry("of multiple application NS source with multiple pipelines", goLangException, func(framework *functional.CollectorFunctionalFramework) {
```

Comment:
Removed as this is covered given input to pipeline to output are isolated so no different then other tests